HDF5Examples/C/H5T/test-pc.sh

```diff
@@ -0,0 +1,250 @@
+#! /bin/sh
+#
+# Copyright by The HDF Group.
+# Copyright by the Board of Trustees of the University of Illinois.
```

Comment:
Probably needs removed as this is a new file and doesn't appear in some of the other new .sh files.